src/utils/crypto.ts

```diff
@@ -0,0 +1,96 @@
+import * as crypto from "crypto";
+import { NonEmptyString } from "@pagopa/ts-commons/lib/strings";
+
+/**
+ * Returns hashed input based on sha256 algo
+ */
+export const toHash = (s: string): NonEmptyString =>
```

Comment:
Maybe `toSha256` is more accurate? @michaeldisaro

Reply:
We changed the fn name to `sha256` to keep the same name used into the `io-backend`.